src/UglyToad.PdfPig.Fonts/SystemFonts/SystemFontFinder.cs

```diff
@@ -89,6 +89,12 @@ static SystemFontFinder()
             {
                 lister = new LinuxSystemFontLister();
             }
+#if NET
```

Comment:
presumably because it could be mono, or some other reason?

Reply:
This was the less restrictive. The IsAndroid check does not exist for framework (obviously hehe)